src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CompositorWindow.cs

```diff
@@ -155,13 +155,34 @@ private void CompositeGUI()
                                 .ToArray());
 
                             compositionManager.CaptureDevice = supportedDevices[selectedIndex];
+
+                            if (supportedDevices[selectedIndex] == FrameProviderDeviceType.AzureKinect)
```

Comment:
>if (supportedDevices[selectedIndex] == FrameProviderDeviceType.AzureKinect) [](start = 28, length = 75)

At some point in the future we might want to have this be a call into the UnityCompositorInterface to ask for something like GetSupportedOcclusionModes(FrameProviderDeviceType type) so that the plugin owns the decision about whether this is supported or not.

Reply:
Makes sense, will add as todo for when we push this back into the spectator view repo